index.md

```diff
@@ -34,6 +34,7 @@ The ARINC653 execution environment (APEX) is under development.
     9. [Running system on `riscv64-generic-qemu`](quickstart/riscv64-generic-qemu.md)
     10. [Running system on `riscv64-generic-spike`](quickstart/riscv64-generic-spike.md)
     11. [Running system on `sparcv8leon3-gr716-mini`](quickstart/sparcv8leon3-gr716-mini.md)
+    12. [Running system on `sparcv8leon3-gr712rc-board`](quickstart/sparcv8leon3-gr712rc-board)
```

Comment:
We have to add it also to a toctree in index.md:
```
```{toctree}
:maxdepth: 1
:hidden:

armv7m4-stm32l4x6-nucleo.md
armv7m7-imxrt105x-evk.md
armv7m7-imxrt106x-evk.md
armv7m7-imxrt117x-evk.md
armv7a7-imx6ull-evk.md
armv7a9-zynq7000.md
armv7a9-zynq7000-qemu.md
armv7a9-zynq7000-zedboard.md
armv7a9-zynq7000-zturn.md
armv8r52-mps3an536-qemu.md
ia32-generic-qemu.md
riscv64-generic-qemu.md
riscv64-generic-spike.md
sparcv8leon3-gr716-mini.md
sparcv8leon3-gr712rc-board
```